### PR TITLE
[DATA-281] Update documentation, including a new quickstart

### PR DESCRIPTION
**Pull request by cranti**
Update autogenerated documentation, including a brand new quickstart! 

* All of the existing examples were preserved, but moved into a sub-page.
* I added a warning that the V1 Stream API is no longer under active development, to encourage users to use the V2 API.
* I tried to keep the code examples reasonably concise (while pointing to the "resources" page for more details) -- but let me know if there's something critical I should highlight.

To review this, I would recommend browsing the docs locally. You can look at the artifact on the CircleCI build _or_ you can check out this branch and run:

```
pip install -r requirements/docs.txt
make -C docs clean html
```

Open the `docs/_build/index.html` file in your browser, and it should all work nicely.

I would love feedback on _any_ of the docs, including things that didn't (obviously) change in this PR. I just made a bunch of changes to the docstrings related to streams/stream metadata. Just give me the name of the function/class/module that you're looking at (or quote the docs directly, and I'll find it).

### CODE REVIEW

**Comment by cranti** [comment]
OH! I also changed the theme that is used to render the docs. The formatting is quite different, but I find the spacing/highlighting to be much more readable... LMK if there are any objections

**Comment by gavin-philips** [comment]
I haven't forgotten this! The tab is sitting here open, waiting for me to get a break from watch issues to review it. Let me know if I'm blocking anything.

**Comment by cranti** [comment]
@trevor-haynes thanks for the (offline) feedback! I added the missing import you flagged + a note about the batch availability API limit:

Screen Shot 2022-09-28 at 12 25 57 PM

@gavin-philips We're hoping to cut the release tomorrow, so I'd love feedback if you have time today! But this can & will evolve after this PR is merged... it's never too late :-)

File: docs/conf.py
Comment (trevor-haynes) [inline docs/conf.py:50]: love the new theme - so much more readable...

**Review by trevor-haynes — APPROVED** [review]
Did a run through from scratch on my own, provided some offline feedback which @cranti has reflected here. Not seeing anything else needing attention. It's a simple workflow with clear instructions. Great job!!

**Review by danirune — APPROVED** [review]

File: docs/pages/quickstart.rst
Comment (gavin-philips) [inline docs/pages/quickstart.rst:186]: This is a nice set of examples, and I love the in_last_two_weeks function. It seems like a good fit for the quickstart.

File: docs/pages/quickstart.rst
Comment (gavin-philips) [inline docs/pages/quickstart.rst:235]: This seems like the perfect end point, drop your data into a dataframe.

File: docs/pages/config.rst
Comment (gavin-philips) [inline docs/pages/config.rst:4]: I like the three examples for different ways to config with tokens. This is often a point of confusion, but with these improved SDK docs, this will be easier to point to, and expect users to read.

File: docs/pages/resources.rst
Comment (gavin-philips) [inline docs/pages/resources.rst:15]: I like that this mirrors the three examples on the Config page.

File: docs/pages/resources.rst
Comment (gavin-philips) [inline docs/pages/resources.rst:64]: Is it possible to add a note on what an org is somewhere around here? Just something simple, like, "A research lab or clinical site is typically represented as an organization."

File: docs/pages/resources.rst
Comment (gavin-philips) [inline docs/pages/resources.rst:75]: If users are confined to working within their current org, what is OrgSet for them? Is this for future functionality?

File: docs/pages/stream.rst
Comment (gavin-philips) [inline docs/pages/stream.rst:10]: I appreciate the wording to steer them away from v1. Will this not be released until all data (events/spans, Percept, etc.) are available from v2? Or are they going to read through this language to do something we're actually directing them to do? And maybe not a question for here, but I don't even know if we'll ever get Summit RC+S into v2...

**Review by gavin-philips — APPROVED** [review]
I love it! Mostly just positive comments, with one or two very minor questions.

This gets me more excited for the SDK, and to show SDK docs to users. Previously, the API docs and example notebooks carried the most weight.

File: docs/pages/quickstart.rst
Comment (cranti) [reply docs/pages/quickstart.rst:90]: Yeah, I'd love to figure out how to describe this more cleanly... there's definitely a little bit of a learning curve to working with V2 data. Let me know if/how folks get stuck!

File: docs/pages/quickstart.rst
Comment (cranti) [reply docs/pages/quickstart.rst:160]: yup & good point! i have to open a follow up PR to bump the library version -- I'll fix it there

File: docs/pages/resources.rst
Comment (cranti) [reply docs/pages/resources.rst:64]: yup! ditto, will fix in the follow up

File: docs/pages/resources.rst
Comment (cranti) [reply docs/pages/resources.rst:75]: yeah, this could be used for users who are members of multiple organizations. we're planning to add the ability to switch your active org via the SDK (instead of requiring a button-click in the browser)

File: docs/pages/stream.rst
Comment (cranti) [reply docs/pages/stream.rst:10]: We are planning to release this now -- but you're right, there is some data that is still only available through the V1 APIs. I'll tweak this in the follow up PR

**Pull request merged**
